WeakAuras/Prototypes.lua

```diff
@@ -1703,7 +1703,7 @@ Private.load_prototype = {
   }
 };
 
-local function AddUnitChangeInternalEvents(triggerUnit, t)
+local function AddUnitChangeInternalEvents(triggerUnit, t, includePets)
```

Comment:
Ah, what was the purpose of this code... I don't quite remember.

Reply:
It's an helper function to set which units WeakAuras.WatchUnitChange should look for and launch scanevents when unit changes faction, role, reaction, or guid.
So for pets, this would track if a pet is mind control or changed guid.